machine/models/machine_instance.py

```diff
@@ -1,16 +1,11 @@
-from odoo import models, fields, api, _
+import ipaddress
+import validators
+
 from odoo.exceptions import ValidationError
 
-PRIORITY = [(1, 'Low'), (2, 'Normal'), (3, 'High')]
-# Machine fields that can be synced.
-SYNC_FIELDS = [
-    'is_virtual',
-    'is_container',
-    'cpu_id',
-    'os_id',
-    'amount_ram',
-    'amount_storage_capacity',
-]
+from odoo import models, fields, api, _
+
+PRIORITY = [('1', 'Low'), ('2', 'Normal'), ('3', 'High')]
```

Comment:
Why not using same strings? Why using number as string is better here?
```suggestion
PRIORITY = [('low', 'Low'), ('normal', 'Normal'), ('high', 'High')]
```

Reply:
Doing same for odootil, also better for migration